standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStore.java

```diff
@@ -4986,6 +4986,10 @@ private void alter_table_core(final String catName, final String dbname, final S
       try {
         Table oldt = get_table_core(catName, dbname, name);
         firePreEvent(new PreAlterTableEvent(oldt, newTable, this));
+        if (newTable.getDbName() == null) {
```

Comment:
In what use case the new DB name can be null?

Reply:
its just for making the exception same ..as per review comment from vary